services/datamanager/builtin/builtin.go

```diff
@@ -197,6 +197,9 @@ func (b *builtIn) Reconfigure(ctx context.Context, deps resource.Dependencies, c
 		// If this error occurs it's a resource graph error
 		return err
 	}
+	if err := os.MkdirAll(captureConfig.CaptureDir, 0o700); err != nil {
```

Comment:
What happens when you call this and the directory already exists?

Reply:
it returns a nil error

Reply:
Cool, thanks for confirming!